apps/climatemappedafrica/src/lib/data/common/index.js

```diff
@@ -57,7 +57,7 @@ function getNavBar(siteSettings, variant, { slug }) {
 export async function getPageProps(api, context) {
   // For now, ClimatemappedAfrica only supports single paths i.e. /, /about, etc.,
   // so params.slug[0] is good enough
-  const slugs = context.params?.slugs || undefined;
+  const slugs = context.params?.slugs || ["index"];
   const [slug] = slugs || ["index"];
```

Comment:
Not sure we need this change. The second line already handles `undefined` with `|| ["index"]`.